### DISCUSSION TITLE
How do I change the alignment of all texts in a page?

### DISCUSSION THREAD

**Discussion by ritog** [quarto-dev/quarto-cli#10586] [Q&A] [answered]
### Description

I am trying to align all the texts of a particular page to left. I can't align all texts to left. Here is the page: [https://ritog.github.io/cv.html](https://ritog.github.io/cv.html). The source can be found [here](https://github.com/ritog/ritog.github.io/blob/master/cv.qmd).

My knowledge of CSS is limited, and I could not do it with anything like:
```html
<div style="text-align: left;">
foo content
</div>
<p style="text-align: left;">
bar content
</div>
```
I also tried basic HTML and CSS. But nothing works.

Can you please tell me how do I do it?

**Comment by mcanouil** [comment, marked as answer]
That's beyond Quarto support.
If you want to customise your HTML, you need to learn (S)CSS, there are no way around that.

Now, one way to solve the issue is to use your favourite browser inspect/developer mode, find the target, create a rule, add that rule to your document/page with the `css` YAML key in your YAML frontmatter (see documentation).

I also suggest you read the documentation about authoring with Quarto, especially the part about using "raw content" and the "divs and spans":
- https://quarto.org/docs/authoring/markdown-basics.html

https://github.com/user-attachments/assets/f465e82a-4739-456a-aad3-6e4b7fe5b3aa

**Reply by ritog** [reply]
Thanks for going out of your way to help me. Appreciate the kindness. The video was great!

The problem was totally resolved.

If you now go to: [https://ritog.github.io/cv.html](https://ritog.github.io/cv.html), it is visible.

I just added the rule for the whole page in a file called **styles.css**, and then added it to **_quarto.yml**:
```yaml
format:
  html:
    css: styles.css
```

And the rule was:

```css
#quarto-document-content {
	text-align: left;
}
```
